ts/Grid/Pro/CellRendering/ContentTypes/SparklineContent.ts

```diff
@@ -0,0 +1,179 @@
+ *
+ *  Authors:
+ *  - Dawid Dragula
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type SparklineRenderer from '../Renderers/SparklineRenderer';
+import type TableCell from '../../../Core/Table/Body/TableCell';
+import type * as HighchartsNamespace from '../../highcharts';
+
+import CellContentPro from '../CellContentPro.js';
+import Globals from '../../../Core/Globals.js';
+import U from '../../../../Core/Utilities.js';
+const {
+    defined,
+    merge
+} = U;
+
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+/**
+ * Represents a checkbox type of cell content.
+ */
+class SparklineContent extends CellContentPro {
+
+
+    public static defaultChartOptions: Globals.DeepPartial<
+    HighchartsNamespace.Options
+    > = {
+            chart: {
+                height: 40,
+                animation: false,
+                margin: [5, 8, 5, 8],
+                backgroundColor: 'transparent'
+            },
+            tooltip: {
+                enabled: false
+            },
+            title: {
+                text: ''
+            },
+            credits: {
+                enabled: false
+            },
+            xAxis: {
+                visible: false
+            },
+            yAxis: {
+                visible: false
+            },
+            legend: {
+                enabled: false
+            },
+            plotOptions: {
+                series: {
+                    borderWidth: 0,
+                    marker: {
+                        enabled: false
+                    },
+                    animation: false
+                }
+            }
+        };
+
+
+    public chart?: HighchartsNamespace.Chart;
+    private chartContainer?: HTMLDivElement;
+
+
+    /* *
+     *
+     *  Constructor
+     *
+     * */
+
+    constructor(cell: TableCell, renderer: SparklineRenderer) {
+        super(cell, renderer);
+        this.add();
+    }
+
+
+    /* *
+     *
+     *  Methods
+     *
+     * */
+
+    protected override add(): void {
+        const renderer = this.renderer as SparklineRenderer;
+        const { chartOptions } = renderer.options;
+        const H = SparklineContent.H;
+        if (!H || !defined(this.cell.value)) {
+            return;
+        }
+
+        this.chartContainer = document.createElement('div');
+        this.cell.htmlElement.appendChild(this.chartContainer);
+
+        this.cell.htmlElement.classList.add(Globals.getClassName('noPadding'));
+
+        let options: Partial<HighchartsNamespace.Options>;
+        if (typeof chartOptions === 'function') {
+            options = chartOptions.call(
+                this.cell,
+                this.cell.value
+            );
+        } else {
+            options = merge(chartOptions) || {};
+        }
+
+        if (!options.series) {
+            options.series = [{
+                data: JSON.parse('' + this.cell.value)
+            }];
+        }
+
+        this.chart = H.Chart.chart(this.chartContainer, merge(
```

Comment:
With the ESM (typical use case in different frameworks), the H is not registered on the window- [demo](https://jsfiddle.net/BlackLabel/97kq1gnL/).

Reply:
It needs to be stated in docs, but with es-modules, Highcharts should be registered in Sparkline Renderers using:
```ts
Grid.CellRenderersRegistry.types.sparkline.useHighcharts(Highcharts);
```

Reply:
Is that a separate task?

Reply:
Yes